cli/src/main/java/org/jboss/as/cli/handlers/ReloadHandler.java

```diff
@@ -220,17 +211,8 @@ private void doHandleEmbedded(CommandContext ctx, ModelControllerClient client)
                     ctx.disconnectController();
```

Comment:
A nit, but the comment at L203 above here is slightly out of date now. The catch of IOException is no longer just to comply with the compiler now that the call may go remote.